### PR TITLE
[zh] Translate kubelet config v1alpha1 reference

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Comment by tengqm** [comment]
Preview: https://deploy-preview-31951--kubernetes-io-main-staging.netlify.app/zh/docs/reference/config-api/kubelet-config.v1alpha1/

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+- If the imageMatch contains a port, then the port must match in the image as well.
+Example values of matchImages:
+  - 123456789.dkr.ecr.us-east-1.amazonaws.com
+  - &lowast;.azurecr.io
+  - gcr.io
+  - &lowast;.&lowast;.registry.io
+  - registry.io:8080/path
+  -->
+<p><code>matchImages</code> 是一个必须设置的字符串列表，用来匹配镜像以便确定是否要调用此提供者。
+如果字符串之一与 kubelet 所请求的镜像匹配，则此插件会被调用并给予提供凭证的机会。
```
Comment (howieyuen) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:212]:
```suggestion
如果列表之一与 kubelet 所请求的镜像匹配，则此插件会被调用并给予提供凭证的机会。
```

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+及对应的详细程度阈值。
+
+## `CredentialProviderConfig`     {#kubelet-config-k8s-io-v1alpha1-CredentialProviderConfig}
+
+<!--
+CredentialProviderConfig is the configuration containing information about
+each exec credential provider. Kubelet reads this configuration from disk and enables
+each provider as specified by the CredentialProvider type.
+-->
+CredentialProviderConfig 包含关于每个 exec 凭据提供者的配置信息。
```
Comment (howieyuen) [inline content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:123]:
```suggestion
CredentialProviderConfig 包含有关每个 exec 凭据提供者的配置信息。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:212]: 我觉得还是表达成“字符串之一”更准确。

Comment (howieyuen) [reply content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md:212]: ok

**Pull request merged**